src/main/java/pl/mmieczak/utils/Utilities.java

```diff
@@ -0,0 +1,27 @@
+package pl.mmieczak.utils;
+
+import pl.mmieczak.model.Category;
+import pl.mmieczak.model.Product;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Utilities {
```

Comment:
Jeżeli już klasa `Utils` to powinna być `final`. Mimo to staraj się nie zabierać logiki z repozytorium. Ono po to właśnie jest żeby mieć odpowiednie metody. Wpadanie w nałóg tworzenia `Utils` bo "może się przydać gdzieś indziej" nie jest dobre :)